src/components/organisms/GlobalSidebar/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { forwardRef } from "react";
+import { BrandLogo } from "@components/molecules/BrandLogo";
+import { XMarkIcon } from "@heroicons/react/24/outline";
+import { NavigationListUI } from "../GlobalNavigation/components/NavigationListUI";
+import { GlobalNavigationProps } from "../GlobalNavigation/type";
+
+type GlobalSidebarProps = {
+  items: GlobalNavigationProps["items"];
+  onClose?: () => void;
```

Comment:
I doesn't really make sense to have it optional, don't you think?
```suggestion
  onClose: () => void;
```

Reply:
@macfrei 
Done!